.github/workflows/esti.yaml

```diff
@@ -326,8 +326,8 @@ jobs:
 
   committed-gc-test:
     name: Test committed gc
-    needs: [ gen-code, deploy-image, login-to-amazon-ecr ]
-    runs-on: ubuntu-20.04
+    needs: [ gen-code, deploy-image, login-to-amazon-ecr, build-spark3-metadata-client ]
+    runs-on: ubuntu-latest-16-cores
```

Comment:
why?

Reply:
We run the other GC tests on 16-core machines, it speeds them up greatly (10 minutes instead of 30)